### PR TITLE
API documentation wording fixes

### PR DESCRIPTION
**Pull request by kirillta**

### CODE REVIEW

**Review by DFSko — APPROVED** [review]

File: HappyTravel.Edo.DirectApi/Models/Booking/BookedRoom.cs [hunk 1]
```diff
@@ -24,13 +24,38 @@ namespace HappyTravel.Edo.DirectApi.Models.Booking
...
+        ///     Deadline and cancellation information
+        /// </summary>
         public Deadline Deadline { get;}
+
+        /// <summary>
+        ///     Booked room type
+        /// </summary>
         public RoomTypes Type { get;}
+
+        // TODO: need more details about the nature of the price
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Booking/BookedRoom.cs:53]: Price for the room (booking price consists of its room prices)

File: HappyTravel.Edo.DirectApi/Models/Booking/Booking.cs [hunk 2]
```diff
@@ -26,17 +26,65 @@ namespace HappyTravel.Edo.DirectApi.Models.Booking
...
+        ///     Cancellation information
+        /// </summary>
         public List<CancellationPolicy> CancellationPolicies { get; }
+
+        /// <summary>
+        ///     Date when an accommodation booking was cancelled. 'NULL' means the booking is not cancelled.
+        /// </summary>
         public DateTime? Cancelled { get; }
+
+        // TODO: the name IsPackageRate also uses
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Booking/Booking.cs:84]: May be renamed to `IsPackageRate`

File: HappyTravel.Edo.DirectApi/Models/Booking/BookingsListFilter.cs [hunk 3]
```diff
@@ -7,6 +7,7 @@ public class BookingsListFilter
     {
         public DateTime? CreatedFrom { get; set; } 
         public DateTime? CreatedTo { get; set; }
+        // TODO: fix naming
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Booking/BookingsListFilter.cs:10]: `CheckInFrom` and `CheckInTo`? ok

File: HappyTravel.Edo.DirectApi/Models/Booking/Deadline.cs [hunk 4]
```diff
@@ -14,9 +14,20 @@ public Deadline(DateTime? date, List<CancellationPolicy>? policies, List<string>
             Remarks = remarks ?? new List<string>(0);
         }
         
-        
+        // TODO: or this is the date when the nearest policy starts?
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Booking/Deadline.cs:17]: Booking cancellation will cost some price after this date.
May be "free cancellation before this date is available"?

File: HappyTravel.Edo.DirectApi/Models/Booking/SlimBooking.cs [hunk 5]
```diff
@@ -21,14 +21,50 @@ namespace HappyTravel.Edo.DirectApi.Models.Booking
...
+        ///     ID for the accommodation
+        /// </summary>
         public string AccommodationId { get; }
+
+        /// <summary>
+        ///     Total net price of a service (This is the <b>actual</b> value for the price)
+        /// </summary>
         public MoneyAmount TotalPrice { get; }
+
+        // TODO: SlimBooking has this field, but Booking hasn't
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Booking/SlimBooking.cs:54]: Let's remove from there if we decided to not use the field on RoomContractSet level

File: HappyTravel.Edo.DirectApi/Models/Booking/SlimBooking.cs [hunk 6]
```diff
@@ -21,14 +21,50 @@ namespace HappyTravel.Edo.DirectApi.Models.Booking
...
+        /// <summary>
+        ///     Indicates if a contract is an advance purchase
+        /// </summary>
         public bool IsAdvancePurchaseRate { get; }
+        /// <summary>
+        ///     Current status of the booking
+        /// </summary>
         public BookingStatuses Status { get; }
+
+        // TODO: we use MainPassenger and IsLeader. THat's inconsistent
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Booking/SlimBooking.cs:64]: Let's use LeaderPassenger?

File: HappyTravel.Edo.DirectApi/Models/Search/DailyRate.cs [hunk 7]
```diff
@@ -12,6 +12,7 @@ namespace HappyTravel.Edo.DirectApi.Models.Search
         public DailyRate(DateTime fromDate, in DateTime toDate, in MoneyAmount finalPrice, in MoneyAmount gross, PriceTypes type,
             string description)
         {
+            // TODO: check nullability
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/DailyRate.cs:15]: Let's check if nullable

File: HappyTravel.Edo.DirectApi/Models/Search/DailyRate.cs [hunk 8]
```diff
@@ -22,37 +23,38 @@ namespace HappyTravel.Edo.DirectApi.Models.Search
...
         /// </summary>
         public DateTime FromDate { get; }
 
         /// <summary>
-        ///     The time frame end date.
+        ///     End of the date range
         /// </summary>
         public DateTime ToDate { get; }
 
+        // TODO: what's the difference between final and total prices?
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/DailyRate.cs:35]: No difference

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs [hunk 9]
```diff
@@ -32,71 +32,75 @@ public record RoomContract
...
         public string MealPlan { get; }
 
+        // TODO: check naming and meaning
         /// <summary>
-        ///     The code of a contract type.
+        ///     Code for the contract type
         /// </summary>
         public string ContractTypeCode { get; }
 
+        // TODO: clarification needed
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:50]: Can be booked right now if true. False for OnRequest rates

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs [hunk 10]
```diff
@@ -32,71 +32,75 @@ public record RoomContract
...
         /// </summary>
         public BoardBasisTypes BoardBasis { get; }
 
         /// <summary>
-        ///     The textual description of a board basis.
+        ///     Description of the board basis (included meals)
         /// </summary>
         public string MealPlan { get; }
 
+        // TODO: check naming and meaning
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:44]: These are contract type codes, in hotel terms, as I know. They may be used to compare the hotel rates from different sources

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs [hunk 11]
```diff
@@ -32,71 +32,75 @@ public record RoomContract
...
+        // TODO: clarification needed
         public bool IsAvailableImmediately { get; }
 
         /// <summary>
-        ///     Indicates if a contract a dynamic offer.
+        ///     Indicates if the contract is a dynamic offer
         /// </summary>
         public bool IsDynamic { get; }
 
+        // TODO: check naming
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:58]: Naming is ok for me

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs [hunk 12]
```diff
@@ -32,71 +32,75 @@ public record RoomContract
...
         [Required]
         public int AdultsNumber { get; }
 
         /// <summary>
-        ///     Ages of each child.
+        ///     Number of children
         /// </summary>
         public int ChildrenNumber { get; }
 
+        // TODO: we pass a database entity straight to the world
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:85]: @DFSko says that this is a separate model. Need to change if not

File: HappyTravel.Edo.DirectApi/Models/Search/WideSearchResult.cs [hunk 13]
```diff
@@ -4,6 +4,7 @@
 
 namespace HappyTravel.Edo.DirectApi.Models.Search
 {
+    // TODO: WideAvailabilityResult and WideSearchResult names are quite similar
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Search/WideSearchResult.cs:7]: Need suggestions, I don't have any

File: HappyTravel.Edo.DirectApi/Models/Static/LocationInfo.cs [hunk 14]
```diff
@@ -27,62 +28,63 @@ namespace HappyTravel.Edo.DirectApi.Models.Static
...
         /// </summary>
         public string CountryId { get; }
 
         /// <summary>
-        ///     The location country name.
+        ///     Name of the country
         /// </summary>
         public string Country { get; }
 
+        // TODO: do we need this?
```
Comment (mingazhev) [inline HappyTravel.Edo.DirectApi/Models/Static/LocationInfo.cs:55]: Don't know, may be can be omitted

File: HappyTravel.Edo.DirectApi/Models/Search/DailyRate.cs (hunk 8 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/Models/Search/DailyRate.cs:35]: In that case, better fit all to the `TotalPrice`

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs (hunk 12 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:85]: > @DFSko says that this is a separate model. Need to change if not

What do you mean by separate?

Comment (mingazhev) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:85]: @DFSko did you lie to me??

Comment (DFSko) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:85]: Dto model created in HappyTravel.Edo.DirectApi.Models.Booking namespace but not used. This is a bug https://github.com/happy-travel/agent-app-project/issues/1190

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs (hunk 10 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:44]: > These are contract type codes, in hotel terms, as I know. They may be used to compare the hotel rates from different sources

I meant here's a _contract_ type code inside a room _contract_

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs (hunk 12 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:85]: image

File: HappyTravel.Edo.DirectApi/Models/Search/WideSearchResult.cs (hunk 13 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/Models/Search/WideSearchResult.cs:7]: ` WideAvailabilitySearchState` or something

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs (hunk 12 above)
Comment (DFSko) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs:86]: Fixed in https://github.com/happy-travel/edo/pull/1439

File: HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs (hunk 10 above)
Comment (kirillta) [reply HappyTravel.Edo.DirectApi/Models/Search/RoomContract.cs]: Leave it be that way so far

**Pull request merged**